docs/whirlpool/docs/03-Getting Started/02-Create Pool/01-Splashpool.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_label: Splash Pool
+---
+
+# Create Splash Pools
+
+Creating a Splash Pool is the easiest way to launch your token on Orca. You only need to provide the mint addresses of 2 tokens and the initial price. It's that simple.
+
+## Function overview
+**`createSplashPoolInstructions()`**
+- **Inputs**:
+
+    - `rpc`: A Solana RPC client used to communicate with the blockchain.
+    - `tokenMintOne` and `tokenMintTwo`: Addresses of the two token mints that will make up the liquidity pool. Selecting which of the two tokens will be token 1 and token 2 matters for the price you are going to set. In most cases, you select your token as token 1 and select SOL/USDC/USDT as token 2.
+    - `initialPrice`: The initial price between the two tokens. You express the value of token 1 in terms of token 2.
+    - `funder`: The account funding the initialization process.
+
+- **Outputs**: The function returns a promise resolving to an object containing:
+
+    - `instructions`: A list of instructions required to initialize the pool.
+    - `initializationCost`: The minimum balance required for [rent](https://solana.com/docs/core/fees#rent) exemption, in lamports.
+    - `poolAddress`: The address of the created pool.
+
+## Basic usage
+
+```tsx title="main.ts"
+import { createSplashPoolInstructions } from '@orca-so/whirlpools-sdk'
+import { Connection, clusterApiUrl } from '@solana/web3.js';
+import { getWallet } from './wallet';
+import { airdropSolIfNeeded } from './airdrop';
+
+async function main() {
+  const connection = new Connection(clusterApiUrl('devnet'));
+  const wallet = getWallet();
```

Comment:
I would not refer back to previous functions you made. Assume that whoever is reading this particular page has not read any of the previous pages.

I would just do two things here instead:
```
const wallet = generateKeypair() // from new web3.js
rpc.requestAirdrop(...) // just always request some airdrop
```
Any ready can always swap this logic out for whatever they need (load from file, etc.)

The quicker you can get to the actual code that matters in this case the easier it becomes to read